<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="graphTimeSelecter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>7 Days</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30 Days</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90 Days</string>
    </property>
   </item>
  </widget>
  <widget class="QGraphicsView" name="graphView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>901</width>
     <height>501</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="g1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="g8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>650</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>571</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>iHaveNoIdea</string>
   </property>
   <widget class="QRadioButton" name="radioComplete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioAverage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioNew">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioTimeSpent">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioCardStrength">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Strength</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
